src/dmd/mtype.d

```diff
@@ -6769,7 +6769,7 @@ extern (C++) final class Parameter : ASTNode
      *      NULL            not found, *pn gets incremented by the number
      *                      of Parameters
      */
-    static Parameter getNth(Parameters* parameters, size_t nth, size_t* pn = null)
+    static Parameter getNth(Parameters* parameters, size_t nth)
```

Comment:
The DDOC comment needs needs to be changed as well.

Reply:
Added proper documentation.